extensions/azurecore/src/account-provider/auths/httpClient.ts

```diff
@@ -284,7 +285,7 @@ const networkRequestViaHttps = <T>(
 ): Promise<NetworkResponse<T>> => {
 	const isPostRequest = httpMethod === HttpMethod.POST;
 	const isPutRequest = httpMethod === HttpMethod.PUT;
-	const body: string = options?.body || '';
+	const body = (new TextEncoder()).encode(JSON.stringify(options?.body || ''));
```

Comment:
So body was an object here instead of a string? Does the msal-common library have the wrong typings or something? It would be good to figure that out and update it if so to avoid other similar problems.

Reply:
ok, I will open an issue on the msal-common library and discuss with the team there.

Reply:
Looked into this and part of the problem is that we define the request body as any here : 

https://github.com/microsoft/azuredatastudio/blob/efd489ecde310661669a2e204486426e4017bf78/extensions/azurecore/src/azureResource/utils.ts#L351-L352

So let's hold off on filing an issue with them since they properly type the body as a string - it's just that using `any` disables the type checking from marking that as an error. I'm looking into an eslint rule to help prevent this in the future.